frontend/src/pages/Login.vue

```diff
@@ -387,45 +387,30 @@ export default {
       );
     },
     showImportView: function() {
-      this.alias = ''
-      this.keystorePath = ''
-      this.keyPasswordValidate = ''
-      this.keystorePassword = ''
-      this.$store.state.walletInfo.keystorePath = ''
-      this.$store.state.walletInfo.alias = ''
-      this.$store.state.walletInfo.keystorePassword = ''
-      this.$store.state.walletInfo.keyPasswordValidate = ''
+      this.resetData();
```

Comment:
This does not reset the form for KeyPass. It actually moves it over to the StorePass form under import and vice versa with the login screen. Super weird. I'm digging into it right now but haven't found anything obvious yet.

Reply:
Also worth noting is that the create new wallet view is unaffected by this. Are you able to reproduce this on your system @digitaltwinnn ?

Reply:
lol yeah I just actually noticed this as well.. code is reused though. The only thing that I can think of is maybe the this.$emit("input", this.password); in Password.vue does something weird

Reply:
It has something to do with the Password component I think, if I comment out the storepass password-input part then the one for keypass behaves as expected also for login->import and vice versa

Reply:
I think I got it, has to with binding stuff into a component I think

Reply:
Amazing! Can't wait to merge this PR :heart_eyes: